src/rantanplan/structures.py

```diff
@@ -0,0 +1,659 @@
+import re
+import statistics
+from collections import Counter
+
+# Stanza structures where each tuple is defined as follows:
+# (
+#     CONSONANT_RHYME | ASSONANT_RHYME,
+#     "structure name",
+#     r".*",  # regular expression to match the rhymed line pattern
+#     lambda lengths: True  # function checking a condition on line lengths
+# )
+# Structures will be checked in order of definition, the first one to match
+# will be chosen.
+
+
+def most_common_lengths(lengths_list, n=4, number_of_verses=None):
+    """Return the n most commons lengths in a list of verses lengths
+
+    :param lengths_list: List with the lengths to be checked
+    :param n: Number of most common lengths to be returned
+    :param number_of_verses: How many verses are in a given stanza
+    :return: The n most common lengths
+    """
+    most_common_lengths_list = []
+    count = Counter(lengths_list).most_common(n)
+    if len(lengths_list) == number_of_verses or number_of_verses is None:
```

Comment:
The line `count = Counter(lengths_list).most_common(n)` can be placed inside the `if` statement.

Reply:
Done